observer.py

```diff
@@ -34,7 +34,8 @@ def get_publishers(network):
 
 
 async def main(args):
-    symbols_to_ignore = set(args.ignore) if args.ignore else set()
+    symbols_to_ignore = set(args.ignore_symbols) if args.ignore_symbols else set()
+    events_to_ignore = set(args.ignore_events) if args.ignore_events else set()
```

Comment:
I think this parameterization isn't flexible enough. We have the following use cases:
1) suppress all alerts for a symbol (e.g., ORCA)
2) suppress a specific error code for a symbol (offline for FX pairs)

One somewhat hacky way to do this is to take a list of regexes. Each regex gets matched against the symbol name + "/" + error code. Then you can specify like `Crypto.ORCA.*` to block all orca alerts, or `FX.*/publisher-offline.*` for the second use case.